Resources/Prototypes/_Moffstation/PersonalItems/Wearables/Pixel8/Marvin_Dugmore/marvinheadphones.yml

```diff
@@ -0,0 +1,32 @@
+- type: entity
```

Comment:
This can be dramatically reduced by inheriting from the existing headphones.
See https://github.com/moff-station/moff-station-14/pull/435/files#diff-045bf4d1c9efc47983365d2ef39f566436d71bec712cb7c059d68300d209b94cR1 for a 1:1 example of how to do that.